apiv2/src/plan-marketing/infra/provider/PlanMarketingBrevo.provider.ts

```diff
@@ -84,6 +84,37 @@ export class PlanMarketingBrevoProvider implements PlanMarketingGateway {
         }
     }
 
+    async deleteOldestListeDiffusion(): Promise<void> {
+        try {
+            // 1. Récupérer toutes les listes
+            const listsResponse = await this.contactsApi.getLists(undefined, undefined, "asc");
+            const listeDiffusions = listsResponse.body.lists;
+            if (!listeDiffusions || listeDiffusions.length === 0) {
+                this.logger.warn("Aucune liste trouvée.");
```

Comment:
ce n'est pas un warning, au contraire c'est un success, tout c'est bien passé !
```suggestion
                this.logger.log("Aucune liste pouvant être supprimé n'a été trouvée.");
```